<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Comparison Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 上传图像按钮 -->
   <widget class="QPushButton" name="button_upload_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_upload_image2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Image 2</string>
    </property>
   </widget>
   <!-- 检材图像显示 -->
   <widget class="QLabel" name="label_image1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 样本图像显示 -->
   <widget class="QLabel" name="label_image2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <!-- 添加点和线 -->
   <widget class="QPushButton" name="button_add_point">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Point</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_add_line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Line</string>
    </property>
   </widget>
   <!-- 撤销按钮 -->
   <widget class="QPushButton" name="button_undo">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Undo</string>
    </property>
   </widget>
   <!-- 截图按钮 -->
   <widget class="QPushButton" name="button_screenshot">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Screenshot</string>
    </property>
   </widget>
   <!-- 保存日志按钮 -->
   <widget class="QPushButton" name="button_save_log">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
</ui>
